gulpfile.babel.js

```diff
@@ -81,49 +74,30 @@ gulp.task('lint', () => {
 
 gulp.task('build', () => {
   return gulp.src('src/*.js')
-          .pipe(errorNotifier())
-
-            .pipe(gulpif(argv.debug, sourcemaps.init()))
-            .pipe(rollup(OPTIONS.rollup))
+    .pipe(errorNotifier())
 
-          .pipe(gulp.dest('.'))
+      .pipe(processJs())
 
-            // Minify and pipe out
-            .pipe(gulpif(!argv.debug, uglify(OPTIONS.uglify)))
-            .pipe(gulpif(argv.debug, sourcemaps.write()))
-            .pipe(rename(path => path.extname = '.min.js'))
-            .pipe(size({ gzip: true }))
+    .pipe(gulp.dest('.'))
 
-          .pipe(gulp.dest('.'))
-          .pipe(notify('build finished'));
```

Comment:
FWIW - this is _really_ helpful, I really want to put this into all our build files actually, or something similar, not having it is a pain because you don't actually know when you can re-run tests. SO you just sit there refreshing the tests browser like a crazy person.